contents/docs/tutorials/toolbar.md

```diff
@@ -186,4 +186,4 @@ That's it! You've now learned to use the main functionalities of our toolbar.
 
 We would like to note once more that the PostHog Toolbar is in Beta (and we mean it!), so if you do find any issues, please report them to us on [GitHub](https://github.com/PostHog/posthog/issues). 
 
-The toolbar should also be getting new features soon, so stay tuned.
\ No newline at end of file
+The toolbar should also be getting new features soon, so stay tuned.
```

Comment:
I'm not sure why this line has been flagged as being changed. It was left unmodified.

Reply:
that is weird!